stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsView.kt

```diff
@@ -76,10 +75,8 @@ internal class MessageOptionsView : FrameLayout {
 
         binding.flagTV.configureListItem(textStyle, style.flagIcon)
         binding.muteTV.configureListItem(textStyle, style.muteIcon)
-        binding.blockTV.configureListItem(textStyle, style.blockIcon)
         binding.editTV.isVisible = false
         binding.deleteTV.isVisible = false
-        configureBlock(configuration = configuration, style = style)
```

Comment:
Shouldn't we remove only `binding.blockTV.configureListItem(textStyle, style.blockIcon)` as it is duplicated, but keep this one? If we remove both, I believe it won't be possible to configure blockTV anymore.

Reply:
👍🏽

Reply:
Ops, it was just moved to the end of the method. Resolving this...